changelog/january2025/2025-01-29-transactional-email-added-automatic-blocklist-for-bette.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title:  Automatic Blocklist for better deliverability
+status: added
+author:
+    fullname: 'Join the #transactional_email channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2025-01-29
+category: managed-services
+product: transactional-email
+---
+
+Optimize your deliverability and improve your reputation score with the automatic blocklist! Addresses that consistently generate hard bounces (mailbox not found, mailbox full) are automatically blocked to prevent them from affecting your email performance.
+
+📖 Learn more 👉 (link doc)
+
```

Comment:
```suggestion

Optimize your deliverability and improve your reputation score with our new automatic blocklist feature. Addresses that consistently generate hard bounces (mailbox not found, mailbox full) are automatically blocked to prevent them from affecting your email performance.

Find out about Transactional Email's blocklist feature in the [dedicated documentation](LINK).
```